<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Menú</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="3" rowspan="15">
         <widget class="QPlainTextEdit" name="firstConsole"/>
        </item>
        <item row="9" column="0" colspan="3">
         <widget class="QSpinBox" name="agregarRed">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="3">
         <widget class="QSpinBox" name="agregarBlue">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="3">
         <widget class="QPushButton" name="agregaFinalButton">
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Agregar Final</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="Consola_de_Dispositivo_Lbl">
          <property name="text">
           <string>Consola de Dispositivo</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QSpinBox" name="agregarDestinoX">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="3">
         <widget class="QSpinBox" name="agregarGreen">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QSpinBox" name="agregarOrigenX">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Origen X</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Destino Y</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="agregarDestinoY">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Origen Y</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="3">
         <widget class="QPushButton" name="agregarInicioButton">
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Agregar Inicio</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2" colspan="2">
         <widget class="QLineEdit" name="agregarID">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QLineEdit" name="agregarVelocidad">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Destino X</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Velocidad</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="agregarOrigenY">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sesión</string>
       </attribute>
       <widget class="QPushButton" name="leerArchivoButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Leer archivo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="guardarSesionButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Guardar sesión actual</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="secondConsole">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>20</y>
          <width>231</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="Consola_de_Dispositivo_Lbl2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Consola de Dispositivo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abrirGraficadorButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Abrir Graficador</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abrirGraficadorButtonButOnlyPointsButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Abrir Graf. pero solo ver Puntos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abrirGraficadorButtonButNearestPoints">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Abrir Graf. Puntos Cercanos -  FB</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <widget class="QTableWidget" name="consolaTabla">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>451</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="barraDeBusqueda">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>370</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="busquedaButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>370</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refrescarButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>370</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Refrescar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Grafo Menu</string>
       </attribute>
       <widget class="QPlainTextEdit" name="thirdConsole">
        <property name="geometry">
         <rect>
          <x>233</x>
          <y>20</y>
          <width>231</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lista de Adyacencia</string>
        </property>
       </widget>
       <widget class="QPushButton" name="generarListaAdyacenciaButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Generar Lista de Adyacencia</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abrirGraficadorButtonKrusk">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Abrir Graf. Algoritmo de Kruskal</string>
        </property>
       </widget>
       <widget class="QPushButton" name="abrirGraficadorButtonPrimm">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Abrir Graf. Algoritmo de Primm</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
